cli/lib/toCubeShape/Dimension.js

```diff
@@ -1,26 +1,31 @@
 const clownface = require('clownface')
 const rdf = require('rdf-ext')
-const TermMap = require('@rdfjs/term-map')
 const TermSet = require('@rdfjs/term-set')
 const ns = require('@tpluscode/rdf-ns-builders')
 
-const datatypeParsers = new TermMap([
-  [ns.xsd.date, term => new Date(term.value)],
-  [ns.xsd.double, term => parseFloat(term.value)],
-  [ns.xsd.float, term => parseFloat(term.value)],
-  [ns.xsd.integer, term => parseInt(term.value)],
-  [ns.xsd.decimal, term => parseFloat(term.value)],
-  [ns.xsd.gYear, term => parseInt(term.value)],
-])
+const datatypeParsers = (datatype) => {
+  switch (datatype.value) {
+    case ns.xsd.date.value:
+      return term => new Date(term.value)
+    case ns.xsd.double.value:
+    case ns.xsd.float.value:
+    case ns.xsd.decimal.value:
+      return term => parseFloat(term.value)
+    case ns.xsd.integer.value:
+    case ns.xsd.int.value:
+    case ns.xsd.gYear.value:
+      return term => parseInt(term.value)
```

Comment:
@tpluscode Not sure if RDF integers/years can start with 0 but just in case they can, it's usually safer to specify the radix explicitly:

```suggestion
      return term => parseInt(term.value, 10)
```

Reply:
When will it make a difference?

Reply:
It used to be a problem because JS would interpret something like `parseInt("010")` as octal instead of decimal. But actually I'm not sure if this is still the case.

https://stackoverflow.com/questions/6611824/why-do-we-need-to-use-radix-parameter-when-calling-parseint

Reply:
I see. MDN mentions that this should be the case when `radix = 0` but it won't hurt to make it explicit an be sure 👌